webpack.config.babel.js

```diff
@@ -7,170 +7,191 @@ import url from 'postcss-url';
 
 const ENV = process.env.NODE_ENV || 'development';
 
-const CSS_MAPS = ENV!=='production';
-
-const baseLoaders = [{
+const baseRules = [{
   test: /\.jsx?$/,
-  include: [`${__dirname}/src`, `${__dirname}/test`],
-  loader: 'babel-loader'
+  include: [
+    `${__dirname}/src`,
+    /*
+    *  Necessary because preact-compat": "3.4.2" has babel in it,
+    *  so webpack2 crashes on UglifyJsPlugin step
+    *  see: https://github.com/developit/preact-compat/issues/155
```

Comment:
Also, the merge functionality is great. Makes creating a bunch of environment dependant config a breeze.